src/Units/KaijuWarsUnits.java

```diff
@@ -727,7 +727,7 @@ public static class GuncrossWing extends KaijuWarsUnitModel
     private static final int UNIT_COST = PREP_COST * 5;
     static final String REZ_TO_NAME = "Guncross Robot";
 
-    private static final MoveType moveType = new Hovercraft();
+    private static final MoveType moveType = new Flight();
```

Comment:
Why are some move types created inside models and others are created once globally (~292-293) and shared?

Reply:
Because I'm a hack and it doesn't really matter anyway - the `UnitModel` constructor is gonna clone whatever it gets passed.
I guess we could clean that up at some point if we care about checking equality on `moveTypes` and/or saving some bytes in our save files.